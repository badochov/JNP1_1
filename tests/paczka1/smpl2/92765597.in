nWRmA0eN4E A639 78,3
nWRmA0eN4E A639 169,1
RdjlFmzE A662 86,8
?
S666 S666 45,6
RdjlFmzE A639 48,0
RdjlFmzE A662 22,1
zvgI5WW1f A639 30,2
?S666
RdjlFmzE A662 18,1
zvgI5WW1f S666 6,3
S666 S666 54,7
vAbJL A347 90,2
zvgI5WW1f S666 78,1
?aukW
vNog A662 87,2
vNog A662 238,1
zvgI5WW1f A347 8,1
vNog A639 7,5
?

vNog A662 86,8
vAbJL A347 264,3
?
zvgI5WW1f A347 40,0
RdjlFmzE A639 8,1
RdjlFmzE A347 80,1
vNog A662 73,8
?
RdjlFmzE A347 241,1
zvgI5WW1f A639 49,0
vNog A662 90,0
zvgI5WW1f A639 232,8
?A662
zvgI5WW1f S666 23,2
nWRmA0eN4E S666 27,0
?RdjlFmzE

?RdjlFmzE
zvgI5WW1f S666 56,8