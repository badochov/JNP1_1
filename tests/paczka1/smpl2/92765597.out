nWRmA0eN4E A 90,8
A639 90,8
RdjlFmzE A 4,0
S666 S 9,1
nWRmA0eN4E A 90,8
vNog A 150,9
zvgI5WW1f S 71,8
A639 90,8
A662 154,9
S666 80,9
RdjlFmzE A 4,0
S666 S 9,1
nWRmA0eN4E A 90,8
vAbJL A 174,1
vNog A 150,9
zvgI5WW1f S 71,8
A347 174,1
A639 90,8
A662 154,9
S666 80,9
RdjlFmzE A 4,0
S666 S 9,1
nWRmA0eN4E A 90,8
vAbJL A 174,1
vNog A 163,9
zvgI5WW1f A 31,9 S 71,8
A347 206,0
A639 90,8
A662 167,9
S666 80,9
A662 167,9
RdjlFmzE A 165,0
RdjlFmzE A 165,0
